pkg/controllers/provisioner/v1alpha1/allocation/allocator.go

```diff
@@ -0,0 +1,69 @@
+/*
```

Comment:
Shall we rename the file to `greedyAllocator.go`

Reply:
It looks like either snake_case or concatcase is the most common: https://github.com/golang/go/issues/36060. I'm partial to the single word file names. Wdyt about greedyallocator.go?